src/components/editor/faultTree/shapes/RenderTree.tsx

```diff
@@ -58,11 +58,14 @@ const renderTree = async (container, node, parentShape = null, pathsToHighlight)
   // For now it seems impossible to detect when operational failure rate was selected.
   // "has" function from lodash can only check property existence with direct path. So we can't show "(o)" in front of "probability"
 
+  // if (has(node, "probability")) {
+  //   nodeShape.attr(
+  //     ["probabilityLabel", "text"],
+  //     `${has(node, "selectedEstimate") ? "(p)" : "(m)"}${node.probability.toExponential(2)}`,
+  //   );
+  // }
```

Comment:
We need to create a separate issue for that.

Reply:
New issue: https://github.com/kbss-cvut/fta-fmea-ui/issues/410